Python/Github_Bomb_Issues/bomb-issues.py

```diff
@@ -0,0 +1,14 @@
+from github import Github
+import os
+import getpass
+
+username = input("Enter your username: ")
```

Comment:
Instead of passing them manually, can you possibly make use of Argument Parsers to pass the credentials as flags

Reply:
Okay!

Reply:
@HarshCasper Should I create another branch and then create another pull request?

Reply:
I guess you can do it in this branch itself. You can squash your Commits before merging them.